test/reuse/util/userSettings/setDateFormatFromUserSettings.spec.js

```diff
@@ -0,0 +1,35 @@
+const data = require("./data/data.json");
+
+describe("userSettings", function () {
+  describe("setDateFormatFromUserSettings.spec - passing date format", function () {
+
+    it("Preparation: Set systemUrl ", async function () {
+      browser.config.params.systemUrl = util.data.decrypt(data.systemUrl);
+    });
+
+    it("Execution & Verification: Set User Date", async function () {
+      await util.userSettings.setDateFormatFromUserSettings(util.data.decrypt(data.username), util.data.decrypt(data.password));
+      const date = await common.date.getToday("yyyy/mm/dd");
+      const userDate = await common.date.getToday(process.env.USER_SETTINGS_DATE_FORMAT);
+      //Ensure returned date matches 10 digits
+      common.assertion.expectEqual(date.length, 10);
```

Comment:
got it, i can replace it with a meaningful constant - it is 10 digits because of the given date formated passed in to the getToday (yyyy/mm/dd), so it should return sth like 2024/08/02 which is 10 chars long.